tools/Makefile

```diff
@@ -21,11 +21,46 @@ ifneq ($(CONFIG_SDK)$(CONFIG_PACKAGE_kmod-b43)$(CONFIG_BRCMSMAC_USE_FW_FROM_WL),
   BUILD_B43_TOOLS = y
 endif
 
-tools-y += autoconf autoconf-archive automake bc bison cmake cpio dosfstools
-tools-y += e2fsprogs expat fakeroot findutils firmware-utils flex gengetopt
-tools-y += libressl libtool lzma m4 make-ext4fs meson missing-macros mkimage
-tools-y += mklibs mtd-utils mtools ninja padjffs2 patch-image
-tools-y += patchelf pkgconf quilt squashfskit4 sstrip zip zlib zstd
+tools-y += \
+	autoconf \
+	autoconf-archive \
+	automake \
+	bc \
+	bison \
+	cmake \
+	cpio \
+	dosfstools \
+	e2fsprogs \
+	expat \
+	fakeroot \
+	findutils \
+	firmware-utils \
+	flex \
+	gengetopt \
+	libressl \
+	libtool \
+	lzma \
+	m4 \
+	make-ext4fs \
+	meson \
+	missing-macros \
+	mkimage \
+	mklibs \
+	mtd-utils \
+	mtools \
+	ninja \
+	padjffs2 \
+	patch-image \
+	patchelf \
+	pkgconf \
+	quilt \
+	squashfskit4 \
+	sstrip \
+	zip \
+	zlib \
+	zstd \
```

Comment:
I'd suggest dropping `\` from the last entry, I believe it's how most our projects do that.

Reply:
the reason I kept it (and the additional newline; is that it allows you to append things cleanly as well, without causing a diff on that. So I'd argue "it still makes sense to do" but if we feel very strongly on not doing it, I don't mind removing it.

Reply:
It's how it's handled in ~99% of our `Makefile`s, please check:
```
grep -E -A 2 ' \\$' tools/*/Makefile
```
So I'd suggest just staying consistent with existing coding style here.

Reply:
Though one could argue that this would be an improvement and we should do it everywhere  :D kind of like how you add the final comma in an enumeration/struct in C ;)

Reply:
To put it in the same record, I'd suggest:
```
tools-y += ninja
tools-y += make #hoho, we can comment now without breaking lines
tools-y += extra # and we don't need to ensure there's a final blank line
tools-y += lolicats # or have diffs always add the \ on the prior last line...
```